spynnaker/pyNN/models/neuron/synapse_dynamics/abstract_synapse_dynamics.py

```diff
@@ -223,6 +223,17 @@ def is_combined_core_capable(self) -> bool:
         """
         raise NotImplementedError
 
+    @property
+    @abstractmethod
+    def is_split_core_capable(self) -> bool:
```

Comment:
first thought is to combine this with is_combined_core_capable as otherwise both could return False

second though is if a model defines both as False the developers should be laughed at

Reply:
This is a tricky one, as both can be True or one can be True, but both can't be False.  So the two are independent but also you need one to be True as otherwise nothing happens.  I have added a check when splitting happens so hopefully that will avoid the situation